scipy/optimize/tests/test_hessian_update_strategy.py

```diff
@@ -30,38 +31,54 @@ def grad(self, x):
         xm_m1 = x[:-2]
         xm_p1 = x[2:]
         der = np.zeros_like(x)
-        der[1:-1] = (200 * (xm - xm_m1**2) -
-                     400 * (xm_p1 - xm**2) * xm - 2 * (1 - xm))
-        der[0] = -400 * x[0] * (x[1] - x[0]**2) - 2 * (1 - x[0])
-        der[-1] = 200 * (x[-1] - x[-2]**2)
+        der[1:-1] = (200 * (xm - xm_m1 ** 2) -
+                     400 * (xm_p1 - xm ** 2) * xm - 2 * (1 - xm))
+        der[0] = -400 * x[0] * (x[1] - x[0] ** 2) - 2 * (1 - x[0])
+        der[-1] = 200 * (x[-1] - x[-2] ** 2)
         return der
 
     def hess(self, x):
         x = np.atleast_1d(x)
         H = np.diag(-400 * x[:-1], 1) - np.diag(400 * x[:-1], -1)
         diagonal = np.zeros(len(x), dtype=x.dtype)
-        diagonal[0] = 1200 * x[0]**2 - 400 * x[1] + 2
+        diagonal[0] = 1200 * x[0] ** 2 - 400 * x[1] + 2
         diagonal[-1] = 200
-        diagonal[1:-1] = 202 + 1200 * x[1:-1]**2 - 400 * x[2:]
+        diagonal[1:-1] = 202 + 1200 * x[1:-1] ** 2 - 400 * x[2:]
         H = H + np.diag(diagonal)
         return H
 
 
 class TestHessianUpdateStrategy(TestCase):
 
     def test_hessian_initialization(self):
-        quasi_newton = (BFGS(), SR1())
+        ndims = 5
+        rnd_matrix = np.random.randint(1, 50, size=(ndims, ndims))
+        init_scales = (('auto', np.eye(ndims)),
+                       (2, np.eye(ndims) * 2),
+                       (np.array(range(1, ndims + 1)), np.array(range(1, ndims + 1)) * np.eye(ndims)),
```

Comment:
Any reason not to use `np.arange()` directly here?

Reply:
Yep, oversight, changed. Thanks!